requirements.txt

```diff
@@ -1,2 +1,2 @@
 click==6.2
-Jinja2>=2.8
+Jinja2==2.8
```

Comment:
Because we know it is working with 2.8, we don't know if it will work with any later version.